src/errors.js

```diff
@@ -60,6 +60,15 @@ class MoleculerRetryableError extends MoleculerError {
 	}
 }
 
+
+class BrokerDisconnectedError extends MoleculerRetryableError {
```

Comment:
Please add tests for this new Error, as well into https://github.com/moleculerjs/moleculer/blob/master/test/unit/errors.spec.js